src/units.py

```diff
@@ -11,13 +12,20 @@ class Units(cfunits.Units):
     class of cfunits to isolate dependence of the framework on cfunits to this
     module.
     """
-    pass
+    def reftime_equivalent(self, other):
```

Comment:
Is this always the case?  I vaguely recall instances where unqualified units were equal to "2000-01-01"

Reply:
Couldn't find any mention of this in the cfunits/cftime docs, but just to be safe I wrote unit tests that checked for this (and other cases) in commit d1435ef9f855879843751a7b0f80d3126445ec3c.